src/netlify_eventobserver.ts

```diff
@@ -61,14 +63,14 @@ netlifyEvents.on('error', ({ branch, context }) => {
   logOutputMessage(`Failed to deploy vscode.ThemeColor${branch} to ${context}`);
```

Comment:
@martinwheeler Do you mind removing `vscode.ThemeColor` from this line? I must have pasted it there by accident :sweat_smile:

Reply:
Hey @ShailenNaidoo this is now done. Sorry it took so long I had a very long break over the holiday period.